service_test.go

```diff
@@ -687,7 +687,7 @@ func waitOrFatal(ch chan bool, t *testing.T) {
 	select {
 	case _ = <-ch:
 		return
-	case <-time.After(500 * time.Millisecond):
+	case <-time.After(1500 * time.Millisecond):
```

Comment:
@jeffallen why this change? is it because travis is slow?

Reply:
Maybe? It was an experiment. I'm not satisfied with it, but it is hard to see what else to do there.